app/services/github_handler.rb

```diff
@@ -1,65 +1,18 @@
 class GithubHandler
-  RETURNS = [
-    SUCCESS = :success,
-    FORBIDDEN = :forbidden,
-    BAD_REQUEST = :bad_request
-  ].freeze
-
   attr_reader :event, :payload, :signature
 
   def initialize(event, payload, signature)
     @event = event
-    @payload = payload
+    @payload = JSON.parse(payload)
     @signature = signature
   end
 
   def handle
-    return FORBIDDEN unless webhook_verified?
-
-    case event
-    when 'pull_request'
-      pull_request_event
-    else
-      BAD_REQUEST
-    end
-    SUCCESS
-  rescue ActiveRecord::RecordInvalid
-    BAD_REQUEST
-  end
-
-  def pull_request_event
-    case payload['action']
-    when 'opened'
-      opened
-    when 'review_requested'
-      review_request
-    when 'closed'
-      closed
-    when 'review_request_removed'
-      review_removal
-    else
-      BAD_REQUEST
-    end
+    Event.create!(data: payload, name: event).handle if webhook_verified?
```

Comment:
Isn't `handle` a private method?

Reply:
Yep, is `resolve` no `handle`, thanks for the correction!